express-stock-trading-platform/server086.js

```diff
@@ -0,0 +1,46 @@
+const express = require('express');
+const app = express();
+
+app.listen(3000, function() {
+  console.log('Your app is listening on port 3000.');
+});
+
+app.get('/hi', (req, res) => {
+  res.send('Hi there trader!');
+});
+
+const prices = {
+  ABC: 48.83,
+  DEF: 2.98,
+  GHI: 3.99,
+  JKL: 99,
+  MNO: 45.38,
+  OPQ: 0.48,
+  RST: 9.32,
+  UVW: 10.94,
+  XYZ: 5.32
+};
+
+app.get('/buy/:ticker/:shares', (req, res) => {
+  const ticker = req.params.ticker;
+  const shares = req.params.shares;
+
+  const total = shares * prices[ticker];
+
+  res.send(
+    `Transaction complete, you purchased ${shares} shares of ${ticker} at $${
+      prices[ticker]
+    }/share for a total of $${total}.`
+  );
+});
+
+app.get('/sell/:ticker/:shares', (req, res) => {
+  const ticker = req.params.ticker;
+  const shares = req.params.shares;
+  const total = shares * prices[ticker];
+});
+
+// Send a response using template strings saying:
```

Comment:
// Send a response using template strings saying:
// `Transaction complete, you sold ___ shares of ___ at $___/share for a total of $___.`

 // Replace the ___ blanks with the appropriate JavaScript variables and be sure to wrap them with `${}`.